### PR TITLE
Move RapidJSON to third_party

### PR DESCRIPTION
**Pull request by klecki**
Signed-off-by: Krzysztof Lecki <klecki@nvidia.com>

### CODE REVIEW

**Comment by klecki** [comment]
!build

File: cmake/Dependencies.cmake [hunk 1]
```diff
@@ -208,4 +208,9 @@ include_directories(${PROJECT_SOURCE_DIR}/third_party/turing_of)
...
 # Boost prerocessor
 ##################################################################
-include_directories(${PROJECT_SOURCE_DIR}/third_party/boost/preprocessor/include)
\ No newline at end of file
+include_directories(${PROJECT_SOURCE_DIR}/third_party/boost/preprocessor/include)
+
+##################################################################
+# RapidJSON
+##################################################################
+include_directories(${PROJECT_SOURCE_DIR}/third_party/rapidjson/include)
```
Comment (szalpal) [inline cmake/Dependencies.cmake]:
```suggestion
include_directories(${PROJECT_SOURCE_DIR}/third_party/rapidjson/include)

```

**Comment by klecki** [comment]
!build

File: cmake/Dependencies.cmake (hunk 1 above)
Comment (klecki) [reply cmake/Dependencies.cmake]: Done

**Comment by klecki** [comment]
!build

**Pull request merged**